src/model.ts

```diff
@@ -14,3 +16,11 @@ export interface Project {
   contributorsCount: number,
   owner: string
 }
+
+export const READY_STATE_DESCRIPTION = {
+  [ReadyState.CONNECTING]: 'Connecting',
+  [ReadyState.OPEN]: 'Open',
+  [ReadyState.CLOSING]: 'Closing',
+  [ReadyState.CLOSED]: 'Closed',
+  [ReadyState.UNINSTANTIATED]: 'Uninstantiated',
+};
```

Comment:
Okay, previously I meant you can use `ReadyState[state]` instead of (as here) `READY_STATE_DESCRIPTION[state]` to convert enum to string. That's what you want to achieve and would be enough for this demo (the text is uppercase however). Just pointing out, no need to change if you want.